features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -503,6 +525,13 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ selectedGrantTypes, isGrantChanged ]);
 
+    useEffect(() => {
```

Comment:
Move the useEffects above the functions.

Reply:
Since we have followed the same approach for other useEffects, shall we keep this as it is?